.env

```diff
@@ -1 +0,0 @@
-VUE_APP_PATH = businesses/edit
```

Comment:
Should this file be deleted?

Reply:
Never mind! GH showed me the old contents so I thought the file still existed.

Reply:
Yes, it will generate during deployment. Or, developer create it own version. Don't need to put into the repo.